samples/bluetooth/central_smp_client/README.rst

```diff
@@ -1,13 +1,13 @@
 .. _bluetooth_central_dfu_smp:
```

Comment:
Update this name?

Reply:
I considered this. But the change will cause problems with building old release notes.